time_tracker_api/api.py

```diff
@@ -4,6 +4,9 @@
 from faker import Faker
 from flask import current_app as app
 from flask_restplus import Api, fields
+from flask_restplus._http import HTTPStatus
+from flask_restplus.fields import Raw, to_marshallable_type, MarshallingError
+from werkzeug.exceptions import Conflict
```

Comment:
I don't see how the two last imports are being used.

Reply:
Sure,  I will remove them